common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -829,6 +829,7 @@ def verify_teams_discussion_permissions(self, should_have_permission):
         self.assertEqual(self.team_page.discussion_id, self.teams[0]['discussion_topic_id'])
         discussion = self.team_page.discussion_page
         self.assertTrue(discussion.is_browser_on_page())
+        discussion.wait_for_element_presence(".discussion", "Waiting on discussion element", 30)
```

Comment:
Definitely agree with putting the css and the waits in the page object.
Number 1 [Test Design Guideline](http://bok-choy.readthedocs.org/en/latest/guidelines.html). :)